test/integration/bodyDictionary.spec.ts

```diff
@@ -33,16 +33,55 @@ describe("BodyDictionary", () => {
       "1": [],
       "2": ["7", "8", "9"]
     });
+    assert.deepEqual(_response.status, 200);
   });
 
+  // FIXME we should de-serialize the incoming value as a null
```

Comment:
Is this comment still valid? If so, can you please add an Issue number so that we can keep track?

Reply:
Yes, I put a link to this issue: https://github.com/Azure/azure-sdk-for-js/issues/12009.